### PR TITLE
feat(feedback): spam detection

### PR DESCRIPTION
**Pull request by JoshFerge**
- Adds llm based spam detection field to issue evidence if the algorithm returns that the message is spam. behind a feature flag.

### CODE REVIEW

**Review by michellewzhang — APPROVED** [review]
nice!! looks like we're just adding a flag `is_spam` onto the evidence, but not doing anything with it yet. will be good to test the behavior before we do anything else

gif

File: tests/sentry/feedback/usecases/test_create_feedback.py [hunk 1]
```diff
@@ -421,3 +440,100 @@ def test_create_feedback_filters_no_contexts_or_message(
...
+            evidence.value
+            for evidence in mock_produce_occurrence_to_kafka.call_args.kwargs[
+                "occurrence"
+            ].evidence_display
+            if evidence.name == "is_spam"
+        ]
+        found_is_spam = is_spam_evidence[0] if is_spam_evidence else None
+        assert (
+            found_is_spam == expected_result
+        ), f"Expected {expected_result} but found {found_is_spam} for {input_message} and feature flag {feature_flag}"
```
Comment (michellewzhang) [inline tests/sentry/feedback/usecases/test_create_feedback.py:539]: probably wanna get rid of this message - looks like it's causing the test to fail?

Comment (JoshFerge) [reply tests/sentry/feedback/usecases/test_create_feedback.py:539]: the message is simply stating why the test failed

Comment (michellewzhang) [reply tests/sentry/feedback/usecases/test_create_feedback.py:539]: ohh wait i saw the red x on the CI test but it's not an actual failure - my bad

File: src/sentry/feedback/usecases/create_feedback.py [hunk 2]
```diff
@@ -169,11 +177,23 @@ def create_feedback_issue(event, project_id, source: FeedbackCreationSource):
     if should_filter_feedback(event, project_id, source):
         return
 
+    project = Project.objects.get_from_cache(id=project_id)
+
+    is_message_spam = None
+    if features.has("organizations:user-feedback-spam-filter-ingest", project.organization):
+        try:
+            is_message_spam = is_spam(event["contexts"]["feedback"]["message"])
```
[the remark this answers is not in the document]
Comment (JoshFerge) [reply src/sentry/feedback/usecases/create_feedback.py:185]: this check is handled by `should_filter_feedback` above

**Pull request merged**